packages/browser/src/utils/lazyLoadIntegration.ts

```diff
@@ -33,12 +33,15 @@ const WindowWithMaybeIntegration = WINDOW as {
 export async function lazyLoadIntegration(name: keyof typeof LazyLoadableIntegrations): Promise<IntegrationFn> {
   const bundle = LazyLoadableIntegrations[name];
 
-  if (!bundle || !WindowWithMaybeIntegration.Sentry) {
+  // `window.Sentry` is only set when using a CDN bundle, but this method can also be used via the NPM package
```

Comment:
Maybe I'm missing some context here but since we never set `window.Sentry` in NPM mode, should we really start doing it now? Just wondering since we have the `__SENTRY__` carrier object anyway 🤔

Reply:
The problem is that the integration CDN bundles _always_ add to `window.Sentry`, so we can't really influence this here (unless we change the logic in our CDN bundles, which is also possible of course 🤔 )

Reply:
Yeah, but even then we'd put the integration onto that object, which is not ideal but should generally also not break stuff (hopefully xD)